packages/plugins/content/slate/src/Component/index.js

```diff
@@ -91,32 +88,30 @@ class Slate extends Component {
     const { document } = serializer.deserialize(data.html)
 
     return state
-      .transform()
+      .change()
       .insertFragment(document)
-      .apply()
   }
 
   render() {
     const {
       focused,
       readOnly,
       state: { editorState },
-      schema,
       plugins,
       onKeyDown,
       HoverButtons,
       ToolbarButtons,
       focus
     } = this.props
-    const isOpened = editorState.isExpanded && editorState.isFocused
+    const isOpened = editorState.selection.isExpanded && editorState.isFocused
 
     return (
       <div>
-        <Portal isOpened={isOpened} onOpen={this.handleOpen}>
+        <Portal onOpen={this.handleOpen}>
```

Comment:
react-portal v4 still has the `isOpened` property (it's now called `isOpen`). Any particular reason why you removed this and put it as a class down below?

Reply:
I didn't like (visually) how when you click somewhere it fades in nicely. But when you click out, it just disappears instantly. This was it will do nice animation both ways. That's the only reason